dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipTcpInboundConnLink.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2008, 2009 IBM Corporation and others.
```

Comment:
Should this copyright be updated due to the usage of netty now in this class?

Reply:
yup